package.json

```diff
@@ -84,7 +84,7 @@
     "dgx-alt-center": "git+https://bitbucket.org/NYPL/dgx-alt-center.git#master",
     "dgx-feature-flags": "git+https://bitbucket.org/NYPL/dgx-feature-flags.git#master",
     "dgx-react-ga": "git+https://git@bitbucket.org/NYPL/dgx-react-ga.git#master",
-    "dgx-svg-icons": "git+https://git@bitbucket.org/NYPL/dgx-svg-icons.git#master",
+    "dgx-svg-icons": "git+https://git@bitbucket.org/NYPL/dgx-svg-icons.git#update-reset-icon",
```

Comment:
Will change back when the PR of dgx-svg-icons gets approved.

Reply:
Added comments to that PR.